processing/pcap_filter.py

```diff
@@ -159,14 +161,16 @@ def __init__(self, *args, **kwargs):
         self.netbios_ignore_list = set(
             ['petra-pc', 'workgroup', 'msbrowse', 'isatap', 'wpad', 'petra-pc.local'])
 
+        self.network_check_to_see = 2 # count request and response
```

Comment:
does it follow redirects? consider moving into a single dns lookup, you're performing two requests and two lookups here